runhouse/servers/cluster_servlet.py

```diff
@@ -41,8 +56,10 @@ async def __init__(
                     )
             self._last_activity = time.time()
             self._last_register = None
-            thread = threading.Thread(target=self.update_autostop, daemon=True)
-            thread.start()
+            autostop_thread = threading.Thread(target=self.update_autostop, daemon=True)
+            autostop_thread.start()
+
+        self.obj_store = None
```

Comment:
I don't think we need to store a `self.obj_store` here? We should use the global `obj_store` associated with the ClusterServlet that's imported at the top of the file.